src/components/paragraph/index.jsx

```diff
@@ -1,9 +1,30 @@
 import PropTypes from "prop-types";
 
-const Paragraph = ({ children }) => <p className="c-paragraph">{children}</p>;
+const COMPONENT_CLASS_NAME = "c-paragraph";
+
+const Paragraph = ({ children, className, truncationLines }) => (
+	<p
+		style={{ "--paragraph-truncation": truncationLines > 0 ? truncationLines : "none" }}
```

Comment:
The falsy part should be null - as the "none" value is already defaulted in within the Sass where you have the fallback CSS Variable declaration.
Also setting this to `null` ensure inline styles are not being set when they are not needed

```suggestion
		style={{ "--paragraph-truncation": truncationLines > 0 ? truncationLines : null }}
```

Reply:
@matthewroach Updated code along with updated tests.